client/state/data-layer/wpcom-http/test/actions.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { http } from '../actions';
+
+const version = 'query.apiVersion';
+const namespace = 'query.apiNamespace';
+
+describe( '#http', () => {
+	it( 'should set the apiVersion', () => {
+		const request = http( { apiVersion: 'v1' } );
+
+		expect( request ).to.have.deep.property( version, 'v1' );
```

Comment:
I'm preferring this in order to keep the test focused. only check for the thing that's under test and not everything else, also handle cases easily when the properties don't exist ("prop doesn't exist" rather than "threw an error")